data/transition-sites/phe_screen_cancer.yml

```diff
@@ -0,0 +1,10 @@
+---
+site: phe_screen_cancer
+whitehall_slug: public-health-england
+homepage: https://www.gov.uk/topic/population-screening-programmes
+tna_timestamp: 20150506150512
+host: www.cancerscreening.nhs.uk
+options: --query-string folder:id
+homepage_title: 'NHS Cancer Screening Programmes'
+aliases:
+  - cancerscreening.nhs.uk
```

Comment:
We've standardised on no indentation for lists in the YAML in this repo (I have a vague recollection that indentation caused parsing errors at some point). There isn't a check in the build for it (there probably should be).